devlab/Vagrantfile

```diff
@@ -68,6 +75,15 @@ Vagrant.configure(2) do |config|
           thisnode.vm.provision "shell",
             path: "./provision/keys.sh",
             args: [ "--public-key", File.read("#{ENV["HOME"]}/#{options['public_key_path']}").strip() ]
+          if nodedata['installer'] == 'devstack'
+            thisnode.vm.provision "file",
+              source: "./provision/devstack/local_#{nodename}.conf", 
+              destination: "local.conf"
+            thisnode.vm.provision "shell",
+              privileged: false,
+              path: "./provision/devstack/devstack.sh",
+              args: [ "--branch", "stable/#{nodename}" ]
```

Comment:
This may work for juno, but will not work for older releases, because they remove branches with release names, leaving only version numbers, such as '2013.1.1', '2014.2.1', etc.